install.sh

```diff
@@ -3778,24 +3794,22 @@ find_git_hook_templates() {
 ############################################################
 mark_directory_as_target() {
     TARGET="$1"
-    if [ "$TARGET" = "" ]; then
+    if [ -z "$TARGET" ]; then
         return
     fi
 
-    if [ "$2" != "" ]; then
```

Comment:
Adding $2 so early in the function is not correct here. See the function description. The logic to be correct  should in a fashion similar to my solution